run_test.sh

```diff
@@ -29,6 +29,7 @@ find generated -type f -not \( -name "*.expected" -or -name "__init__.py" \) -de
     fi
     $PROTOC --mypy_out=generated --proto_path=proto/ --experimental_allow_proto3_optional `find proto/ -name "*.proto"`
     $PROTOC --python_out=generated --proto_path=proto/ --experimental_allow_proto3_optional `find proto/testproto -name "*.proto"`
+    $PROTOC --mypy_grpc_out=generated --proto_path=proto/ --experimental_allow_proto3_optional `find proto/testproto/grpc -name "*.proto"`
```

Comment:
Idea would be to limit this by an if statement around the python version.

Reply:
this line works fine for all versions

Reply:
I guess I need to add if statement below where mypy reads generated files. not sure how to do it

Reply:
Maybe limit this line by
`if $PY_VER_MYPY_TARGET != 2.7` ?

Also noticing that the previous line (line 38) is also compiling grpc protos based on the structure you've used.
Perhaps grpc test protos should go separately (eg proto/testgrpc?)